v2/cmd/functional-test/run.sh

```diff
@@ -1,13 +1,34 @@
 #!/bin/bash
 
-echo 'Building functional-test binary'
-go build
+# reading os type from flags
+CURRENT_OS=$1
 
-echo 'Building Nuclei binary from current branch'
-go build -o nuclei_dev ../nuclei
+if [ "${CURRENT_OS}" == "windows" ]
+then
```

Comment:
The only difference between the two code blocks is the `.exe` extension. Please save the file name in a variable using the `if` statement and use that instead of duplicating the code.

Reply:
Done